src/pal/src/arch/i386/asmconstants.h

```diff
@@ -63,14 +65,13 @@
 #define CONTEXT_Xmm13 CONTEXT_Xmm12+16
 #define CONTEXT_Xmm14 CONTEXT_Xmm13+16
 #define CONTEXT_Xmm15 CONTEXT_Xmm14+16
-#define CONTEXT_VectorRegister CONTEXT_Xmm15+16
-#define CONTEXT_VectorControl CONTEXT_VectorRegister+16*26
-#define CONTEXT_DebugControl CONTEXT_VectorControl+8
+#define CONTEXT_Ymmh CONTEXT_FltSave+FLOATING_SAVE_AREA_SIZE
+#define CONTEXT_DebugControl CONTEXT_Ymmh+16*16
```

Comment:
This doesn't seem to match the CONTEXT structure defined in pal.h. There is a VectorRegister array of 26 M128A entries and DWORD64 VectorControl after the floating save area there, while you assume only 16 M128A entries here.